stories/assets/scss/_breakpoints.scss

```diff
@@ -5,7 +5,12 @@
       @content;
     }
   }
-
+  @else if $point==mediumonlytab {
+    /* medium design tab */
+    @media (max-width: 89.9375em) and (min-width:48em){
```

Comment:
@Ashish-srij It should be written like @media (min-width:48em) and (max-width: 89.9375em)